README.md

```diff
@@ -6,6 +6,19 @@ The Retinopathy Risk Assessment Tool (RRAT) is a data visualization web applicat
 
 ## Planned Features
 
+## Features
+- Image Upload: Securely upload patient retinal images for analysis.
+- Patient Data Input: Add patient details such as name, date of birth, and medical history to enhance analysis accuracy.
+- Model-Driven Analysis: Utilizes a trained machine learning model to assess the risk of diabetic retinopathy.
+- Results Display: Presents a clear and concise prognosis, helping clinicians make informed decisions.
+- Additional Resources: May add resources and next steps, such as links to Mayo Clinic or NIH, after prognosis.
+
+## Technology Stack
+- Backend: Python (Django)
+- Frontend: Javascript, Material-UI, Tailwind CSS
+- Database: SQLite => MySQL
```

Comment:
For now we are keeping SQLite, so keep it as SQLite for now. (Unless the wizard uses it, then keep it)